cmd/panfs.go

```diff
@@ -691,13 +685,15 @@ func (fs *PANFSObjects) listBuckets(ctx context.Context) ([]BucketInfo, error) {
 		// We need "buckets/" here, not "buckets" to list all the
 		// objects included in the "buckets" directory but ignore a
 		// directory/object whose name would begin with "buckets".
-		entries, err = fs.configAgent.GetObjectsList(panfsBucketListPrefix + SlashSeparator)
+		prefix := pathJoin(minioMetaBucket, bucketMetaPrefix, SlashSeparator)
+		entries, err = fs.configAgent.GetObjectPrefixes(prefix, SlashSeparator)
```

Comment:
There's a directory for each bucket which contains a .minio.sys file. Also replication-related data.

Reply:
As we discussed this in teams, maybe we can just list all objects in 'buckets' then (no prefixes)? As we should have only bucket metadata files. And this is exactly the place where we can use set/map just in case if we ever have some unexpected other files in that location. So in short, I do not think that it's worth sorting prefixes on pan-config side in that case.